# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: testing/web-platform/tests/editing/data/multitest.js
# Commit: 5bacd687fa98
# Full Hash: 5bacd687fa98b0e9a5dea9944f360fdc5a6770e6
# Author: Masayuki Nakano <masayuki@d-toybox.com>
# Date: 2023-01-17 16:13:02
# Regressor Bug: 1807829
# File Overlap Count: 2
# Description:
#   Bug 1807829 - Make `HTMLEditor::CreateStyleForInsertText` use same logic as `HTMLEditor::SetInlinePropertiesAsSubAction` to apply new style r=m_kato
#   
#   The problem in Yahoo mail is, when you apply new font size to:
#   ```
#   <div><font size="7">{}<br></font></div>
# ==============================================================================

diff -r a7465d2d89ba -r 5bacd687fa98 testing/web-platform/tests/editing/data/multitest.js
--- a/testing/web-platform/tests/editing/data/multitest.js	Mon Jan 16 23:14:12 2023 +0000
+++ b/testing/web-platform/tests/editing/data/multitest.js	Tue Jan 17 05:26:14 2023 +0000
@@ -2853,4 +2853,17 @@
     [true,true,true],
     {}],
 
+// <font> element should be reused when the font-size is change for new text.
+["<font size=7>{}<br></font>",
+    [["stylewithcss","false"],["fontsize","4"],["insertText","a"]],
+    ["<font size=\"4\">a[]<br></font>",
+     "<font size=\"4\">a[]</font>"],
+    [true,true,true],
+    {"fontsize":[false,false,"7",false,false,"4"]}],
+["<span style=font-weight:bold>{}<br></span></b>",
+    [["stylewithcss","true"],["italic",""],["insertText","a"]],
+    ["<span style=\"font-weight:bold; font-style:italic\">a[]<br></span>",
+     "<span style=\"font-weight:bold; font-style:italic\">a[]</span>"],
+    [true,true,true],
+    {}],
 ]
